test/verify/check-client

```diff
@@ -143,21 +143,16 @@ exec "$@"
         self.check_login("admin@target")
         b.become_superuser()
 
-        # cockpit-pcp is installed on client, but not on target; recognize that
+        # python3-pcp is installed on client, but not on target; recognize that
         if self.m_target.image != 'arch':
```

Comment:
This should now also work on arch, right? OK to be a follow-up, but this PR changes arch code, so it needs to be tested at least manually, or the follow-up is a release blocker.

Reply:
It should work fine, I'm happy to do this in a follow up

Reply:
Sneaked this in as last commit.